riptide-opentracing/README.md

```diff
@@ -92,6 +97,44 @@ new OpenTracingPlugin(tracer)
     .withAdditionalSpanDecorators(new HttpUrlSpanDecorator())
 ```
 
+### Lifecycle Policy
+
+A lifecycle policy can be used to specify which spans are reused or whether a new one is created:
+
+```java
+new OpenTracingPlugin(tracer)
+    .withLifecyclePolicy(new NewSpanLifecyclePolicy());
+```
+
+| Lifecycle Policy              | Description                                                         |
+|-------------------------------|---------------------------------------------------------------------|
+| `ActiveSpanLifecyclePolicy`   | Reuses the current active span.                                     |
```

Comment:
I think that specifying some samples why people want to use this will help for adoption of this.
From my side - without looking on the code but just on this documentation I have no idea what lifecycle would be interesting to use for me (for common use case like create new span/attach it to existing if present)

Reply:
I added some more docs.